lib/index.js

```diff
@@ -5,10 +5,7 @@ var Resource = require('./Resource'),
     Controllers = require('./Controllers'),
     Errors = require('./Errors'),
     inflection = require('inflection'),
-    _ = require('lodash');
-
-// make deep defaults the default
-_.defaults = require('merge-defaults');
+    mergeDefaults = require('merge-defaults');
```

Comment:
@Fridus @LoicMahieu thanks for the detailed explanations. I hadn't considered that npm was optimizing for us here, and agree this is definitely the way to go.